mesheryctl/internal/cli/root/system/start.go

```diff
@@ -79,174 +85,328 @@ func ValidateComposeFileForRecreation(CurrentServices map[string]utils.Service,
 	return nil
 }
 
-func start() error {
-	if _, err := os.Stat(utils.MesheryFolder); os.IsNotExist(err) {
-		if err := os.Mkdir(utils.MesheryFolder, 0777); err != nil {
-			return errors.Wrapf(err, utils.SystemError(fmt.Sprintf("failed to make %s directory", utils.MesheryFolder)))
-		}
-	}
-
-	if _, err := os.Stat(utils.DockerComposeFile); os.IsNotExist(err) {
-		if err := utils.DownloadFile(utils.DockerComposeFile, fileURL); err != nil {
-			return errors.Wrapf(err, utils.SystemError(fmt.Sprintf("failed to download %s file from %s", utils.DockerComposeFile, fileURL)))
-		}
-	}
+// applyManifest is a wrapper function for client.ApplyManifest
+func applyManifest(manifest []byte, client *meshkitkube.Client) error {
+	// ApplyManifest applies the given manifest file to the Kubernetes cluster
+	err := client.ApplyManifest(manifest, meshkitkube.ApplyOptions{
+		Namespace: "meshery",
```

Comment:
Ideally, the namespace is an externalized configuration item.

Reply:
I agree

Reply:
As in it should not be hard-coded?